docs/Concepts/Node-Types.md

```diff
@@ -37,19 +37,28 @@ genesis block.
 
 !!! important
 
-    Fast synchronization is the default for named networks specified with
-    the [`--network` option](../Reference/CLI/CLI-Syntax.md#network), except for the `dev` development
-    network.
-
-    Fast synchronization is also the default if Ethereum mainnet is being connected to by not specifying
-    the [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
+    Fast synchronization is the default for named networks specified using the
+    [`--network`](../Reference/CLI/CLI-Syntax.md#network) option, except for the `dev` development network.
+    It's also the default if connecting to Ethereum MainNet by not specifying the
+    [`--network`](../Reference/CLI/CLI-Syntax.md#network) and [`--genesis-file`](../Reference/CLI/CLI-Syntax.md#genesis-file)
     options.
 
 !!! note
 
-    To improve performance, or if the fast sync is taking a long time to complete, use
+    When fast synchronizing, block numbers increase until close to the head block, then the synchronization stalls for
+    some time depending on world state size.
```

Comment:
I'm not sure I'd say "stalls". Maybe something like "then the block import process pauses while the world state download completes. This may take a significant amount of time, during which the current head block will not increase."